bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_13/A5_13.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.enocean.internal.eep.A5_13;
+
+import org.openhab.binding.enocean.internal.eep.Base._4BSMessage;
+import org.openhab.binding.enocean.internal.messages.ERP1Message;
+
+/**
+ *
+ * @author Daniel Weber - Initial contribution
+ */
+public abstract class A5_13 extends _4BSMessage {
+    public A5_13(ERP1Message packet) {
+        super(packet);
+    }
+
+    protected final int PARTONE = 0x10;
+    protected final int PARTTWO = 0x20;
+
+    protected int getMessageIdentifier() {
+        return getDB_0Value() & 0xF0;
+    }
+
+    protected boolean isPartOne() {
+        return getMessageIdentifier() == PARTONE;
+    }
+
+    protected boolean isPartTwo() {
+        return getMessageIdentifier() == PARTTWO;
+    }
+}
```

Comment:
Please make sure all your files contain a newline at the end of the file

Reply:
should be solved